src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+Microsoft.Network;microsoft.network/p2svpngateways;VPN Gateway;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/privatednszones;Azure Private Link;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/privateendpoints;Azure Private Link;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/publicipaddresses;Virtual Network;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/publicipprefixes;Virtual Network;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/trafficmanagerprofiles;Traffic Manager;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/virtualhubs;Virtual WAN;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/virtualnetworkgateways;Virtual Network;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/virtualnetworks;Virtual Network;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Network;microsoft.network/vpngateways;VPN Gateway;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.NetworkAnalytics;microsoft.networkanalytics/dataconnectors;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.NotificationHubs;microsoft.notificationhubs/namespaces;Notification Hubs;Mobile;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.nutanix;microsoft.nutanix/nodes;Azure VMWare Solution;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.OpenEnergyPlatform;microsoft.openenergyplatform/energyservices;Azure Data manager for Energy;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/azurefirewall;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/clusters;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/networksecuritygroups;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/p2svpngateways;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/virtualnetworkgateways;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;SaaS
+microsoft.operationalinsights;microsoft.operationalinsights/workspaces;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Orbital;microsoft.orbital/spacecrafts;Azure Orbital;Networking;Microsoft;Cloud Provider;Cloud;SaaS
+MICROSOFT.ORBITAL;microsoft.orbital/terminals;Azure Orbital;Networking;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Peering;microsoft.peering/peerings;Azure Peering Service;Networking;Microsoft;Cloud Provider;Cloud;PaaS

```

Comment:
IaaS?

Reply:
Azure Peering Service can be classified as a Networking as a Service (NaaS). It’s a category of cloud services that provides businesses and individuals with networking resources on-demand. This includes wide area networking (WAN), bandwidth, optimization, network management, virtual private networks (VPNs), and more. Azure Peering Service, as a NaaS offering, enhances the performance and reliability of public internet connectivity to Microsoft’s cloud services. It’s important to note that it doesn’t fall directly under the traditional cloud service models of SaaS, PaaS, or IaaS. Instead, it’s a specialized networking service.

How to classify it then ?

Reply:
I think because this is an initial release, we can keep things a bit more simple and call this one IaaS. We can always come back later and be more precise with the definitions. @flanakin FYI.

Reply:
Classified as IaaS, maybe some precisions in the readme section ?